security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainAdd.java

```diff
@@ -257,12 +257,12 @@ private boolean processAudit(OperationContext context, String securityDomain, Mo
         AuditInfo auditInfo = new AuditInfo(securityDomain);
         for (Property moduleProperty : node.asPropertyList()) {
             ModelNode module = moduleProperty.getValue();
-            String codeName = LoginModuleResourceDefinition.CODE.resolveModelAttribute(context, module).asString();
+            String codeName = ProviderModuleResourceDefinition.CODE.resolveModelAttribute(context, module).asString();
```

Comment:
where does this make sense?
ProviderModuleResourceDefintion is never added? where are parsers? where are tests for that?